clients/java/src/main/java/io/zeebe/client/api/worker/JobClient.java

```diff
@@ -65,4 +66,25 @@
    * @return a builder for the command
    */
   FailJobCommandStep1 newFailCommand(long jobKey);
+
+  /**
+   * Command to throw a non-technical error.
```

Comment:
Using the same wording as in the gRPC proto.

```suggestion
   * Command to report a business error (i.e. non-technical) that occurs while processing a job.
```